packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -183,6 +187,23 @@ export async function runNpmInstall(
       opts
     );
   }
+
+  if (packageJson) {
+    // Add `node_modules/.bin` to the PATH env
+    const { stdout } = hasPackageLockJson
+      ? await execa('npm', ['bin'], { cwd: destPath })
+      : await execa('yarn', ['bin'], { cwd: destPath });
```

Comment:
Why  are you using`execa` instead of the `spawnAsync()` function defined above?

Reply:
@styfle because I need the `stdout` output